pkgs/by-name/na/nanorpc/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost186,
+  openssl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nanorpc";
+  version = "1.1.0-unstable-2021-04-29";
```

Comment:
The actual nanorpc package should probably be upstream's, since it now supports up to C++17 instead of C++14:

https://github.com/luxonis/nanorpc/compare/master..eb363c7004580d711f88ee1785d87ae6fc469220